app/(route)/my/history/[type]/page.tsx

```diff
@@ -77,14 +19,49 @@ const MyHistory = (props: any) => {
 
   const title = type === 'selling' ? '판매' : '구매';
 
-  const historyData = type === 'selling' ? data.selling : data.buying;
-
   const router = useRouter();
-
-  const datePickerValues = [
-    { value: startDate, setValue: setStartDate },
-    { value: endDate, setValue: setEndDate },
+  const { data: bidding, refetch: bidRe } = useProductHistory({
+    type,
+    status: 'bidding',
+    startDate: startDate.format('YYYY-MM-DD'),
+    endDate: endDate.format('YYYY-MM-DD'),
+  });
+
+  const { data: pending, refetch: pendRe } = useProductHistory({
+    type,
+    status: 'pending',
+    startDate: startDate.format('YYYY-MM-DD'),
+    endDate: endDate.format('YYYY-MM-DD'),
+  });
+
+  const { data: finished, refetch: finiRe } = useProductHistory({
+    type,
+    status: 'finished',
+    startDate: startDate.format('YYYY-MM-DD'),
+    endDate: endDate.format('YYYY-MM-DD'),
+  });
```

Comment:
```suggestion
  const formatDate = (date) => date.format('YYYY-MM-DD');

  const statuses = ['bidding', 'pending', 'finished'];
  const dataPromises = statuses.map((status) =>
    useProductHistory({
      type,
      status,
      startDate: formatDate(startDate),
      endDate: formatDate(endDate),
    })
);

const [bidding, pending, finished] = await Promise.all(dataPromises);
```
같은 코드가 여러번 반복되어서 하나로 합치는게 좋을 것 같습니다
위 코드는 챗지피티의 추천 코드이니 참고하세요!